documentcloud/client.py

```diff
@@ -131,12 +126,12 @@ def user_id(self):
     def _request(self, method, url, raise_error=True, **kwargs):
         """Generic method to make API requests"""
         # pylint: disable=method-hidden
-        logger.info("request: %s - %s - %s", method, url, kwargs)
```

Comment:
Logs should be lazy, as pylint is now warning:

https://medium.com/flowe-ita/logging-should-be-lazy-bc6ac9816906